internal/deviations/deviations.go

```diff
@@ -65,8 +65,15 @@ package deviations
 
 import (
 	"flag"
+
+	"github.com/openconfig/ondatra"
 )
 
+// P4RTMissingDelete adds deviation for device that does not support delete mode in P4RT write requests.
+func P4RTMissingDelete(dut *ondatra.DUTDevice) bool {
```

Comment:
I guess let's name the "dut" parameter "_" for now to silence this static analysis

Reply:
done